src/cargo/util/toml/targets.rs

```diff
@@ -38,19 +39,19 @@ pub fn targets(manifest: &TomlManifest,
     }
 
     targets.extend(
-        clean_bins(manifest.bin.as_ref(), package_root, package_name, warnings, has_lib)?
+        clean_bins(manifest.bin.as_ref(), package_root, package_name, warnings, errors, has_lib)?
```

Comment:
Hm, now I am not quite sure that we should apply this logic to binaries as well...